Tasks/DownloadBuildArtifacts/task.json

```diff
@@ -68,9 +68,9 @@
       "helpMarkDown": "If checked, this build task will try to download artifacts from the triggering build. If there is no triggering build from the specified definition, it will download artifacts from the build specified in the options below."
     },
     {
-      "name": "artifactVersion",
+      "name": "buildVersionToDownload",
```

Comment:
folks only see the label. Is there a reason to rename the name?  It will cause the tasks to get redownloaded since I believe the folder is by name + guid.

Reply:
What does this mean to yaml and all our docs and templates?  Is this rename really worth it?

Reply:
This is a naming change of a field that is added as part of this feature (initially implemented by me, named 'artifactVersion'). On top of that, this is only a renaming of this parameter, not the renaming of the whole step. I dont think this is breaking in any way. With this, I dont see how this affects the yaml/docs... Or am I missing something here?